src/types.rs

```diff
@@ -207,8 +208,30 @@ pub struct StreamClaimReply {
 ///
 /// [`xpending`]: ./trait.StreamCommands.html#method.xpending
 ///
+#[derive(Debug)]
+pub enum StreamPendingReply {
+    Empty,
+    Data(StreamPendingData),
```

Comment:
Consider changing this to either `NotEmpty(StreamPending)` or `Pending(StreamPending)`.